src/components/Project/Releases/index.js

```diff
@@ -98,12 +97,18 @@ class ReleaseView extends React.Component {
     if (seconds < 10) {
       seconds = "0" + seconds
     }
-
-    if(parseInt(hours, 10) < 1) {
-      return minutes+':'+seconds;
-    } else {
-      return hours+':'+minutes+':'+seconds;
+    
+    if(parseInt(hours,10) < 1) {
```

Comment:
@drshrey I'd suggest using a 3rd party library for this or extracting it out into a helper function to be used/tested outside of this 1 component.